### DISCUSSION TITLE
Health Check Usage and Expectations Questionnaire

### DISCUSSION THREAD

**Discussion by mmandrus** [grafana/grafana#58100] [General]
Hello everyone! Our team at Grafana is doing some exploration of how we might bolster Grafana's current health checks. While today, Grafana provides basic health endpoints and a lot of logs and metrics that you can use to monitor your instances, we do not provide a simple, out-of-the-box way of answering the question, "is my instance healthy?" We would like to develop a solution to this problem that serves many different types of user.

Please tell us what matters to you by filling out [this form](https://docs.google.com/forms/d/e/1FAIpQLScfpPOjGtvSZaoFdkVjbIGKP-OEVJcF9Y9nqYmRK_RdI7RG6A/viewform?usp=sf_link). Responses will be anonymous. Any additional comments in the discussion are welcome as well. We appreciate any thoughts you have to share with us!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
